src/cdk/drag-drop/drag-drop-registry.ts

```diff
@@ -133,6 +137,7 @@ export class DragDropRegistry<I, C extends {id: string}> implements OnDestroy {
 
     if (this._activeDragInstances.size === 0) {
       this._clearGlobalListeners();
+      this._document.body.classList.remove('cdk-drag-drop-disable-native-interactions');
```

Comment:
Will this still get removed if something interrupts the drag? Something like an OS-level prompt or the OS changing window focus/mouse location.

Reply:
As long as the mouseup/touchend event fires, it will.

Reply:
I'd prefer to avoid adding more timers. Even if the user's pointer leaves the browser and then comes back, the next `mouseup`/`touchend` event will reset it.